src/Azure.Functions.Cli/Common/ExtensionPackage.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Azure.Functions.Cli.Common
 {
-    class ExtensionPackage
+    public class ExtensionPackage
```

Comment:
intentionally making this public?

Reply:
yeah was needed for the enum. may end up removing this after switching to enum from webjobs sdk.